dpos/p2p/hub/conn.go

```diff
@@ -74,6 +75,7 @@ func WrapConn(c net.Conn) (conn *Conn, err error) {
 	}
 
 	if hdr.GetCMD() != p2p.CmdVersion {
+		err = errors.New("invalid CMD version")
```

Comment:
This error should be the opposite, `fmt.Errorf("invalid message %s, expecting version", hdr.GetCMD())`

Reply:
fixed